src/main/java/Bye.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Ends the ChatBot.
+ */
+public class Bye extends Command {
+
+    Bye() {
+        this.name = "bye";
+        this.usage = "bye";
+        this.description = "Duke Chatbot will power down";
+    }
+
+    public String response() {
```

Comment:
Perhaps use `respond()` instead? The coding standard states "Names representing methods must be verbs".